tests/integ/modin/groupby/test_groupby_named_agg.py

```diff
@@ -55,3 +63,74 @@ def test_valid_func_with_named_agg_errors(basic_snowpark_pandas_df):
         assert_exception_equal=False,  # There is a difference in our errors.
         expect_exception_type=TypeError,
     )
+
+
+# Query count is 2 in below test due to line 65, basic_snowpark_pandas_df.to_pandas()
```

Comment:
Line number is variable, so you should omit it.
```suggestion
# Query count is 2 in below test due to basic_snowpark_pandas_df.to_pandas()
```

Reply:
Is it possible to change the fixture to a native pandas frame so we don't do this extra to_pandas()?